eg/Classes/MainFrame/__init__.py

```diff
@@ -521,12 +521,45 @@ def GetPanelDirection(self, panel):
         pos = info.find(";dir=") + 5
         return info[pos]
 
+    def SetWindowStyleFlag(self, *args, **kwargs):
+        """
+        Changes the main frame style depending on the display of the tray icon.
+
+        sets the stlye flags fo the minimize button will be grayed out if the
+        tray icon is shown and there are any open child windows.
+        If the tray icon is not shown the minimize button will function as
+        usual.
+
+        :param args: unused parameter
```

Comment:
so it won't toss out a traceback. I am not sure if the internals of wx use it at all so i wanted to make sure that it wouldn't cause an issue.